react/wbooks/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import i18next from 'i18next';
+
+import styles from './styles.module.scss';
+
+function Login() {
+  return (
+    <div className={styles.container}>
+      <p>{i18next.t('LOGIN:login')}</p>
```

Comment:
you can choose `<div>` or `<p>` and put the styles on it. If you put the styles on one of them, the other is unnecesary.